.github/workflows/timechart.yml

```diff
@@ -20,4 +21,16 @@ jobs:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
         with:
           SHA: "${{ github.event.pull_request.head.sha }}"
-          TRACE_START: "${{ github.event.pull_request.updated_at }}"
\ No newline at end of file
+          TRACE_START: "${{ github.event.pull_request.updated_at }}"
+
+      # It's difficult to determine all jobs for a PR
+      # see https://docs.mergify.io/conditions/#about-status-checks
+      # Since we have this 'checkrun-timechart' job, which essentially
+      # knows all the statuses (see https://github.com/WyriHaximus/github-action-wait-for-status#output),
+      # we can leverage it to provide a global status to mergify.
+      - name: 'mergify status'
+        uses: actions/github-script@v3
+        if: steps.waitforstatuschecks.outputs.status != 'success'
+        with:
+          script: |
+            core.setFailed('One of the tests failed')
```

Comment:
Could this also be something other than a test? I.e. compiling and synthezising an example?

Reply:
better? 8e68df7

Reply:
yes! Besides the typos 😅 (fixed them already)